packages/amplify-category-auth/src/provider-utils/supported-services.ts

```diff
@@ -1164,6 +1176,284 @@ export const supportedServices = {
+        required: false,
+        andConditions: [
+          {
+            key: "authProvidersUserPool",
+            value: "OIDC",
+            operator: "includes"
+          }
+        ],
+        validation: {
+          operator: "regex",
+          value: ".*",
+          onErrorMsg: "The value can be anything."
+        }
+      },
+      {
+        key: "oidcAppOIDCIssuer",
+        question: "Enter your OIDC Issuer url: ",
+        required: true,
+        andConditions: [
+          {
+            key: "authProvidersUserPool",
+            value: "OIDC",
+            operator: "includes"
+          }
+        ],
+        validation: {
+          operator: "regex",
+          value: "^(((?!http://(?!localhost))([a-zA-Z0-9.]{1,})://([a-zA-Z0-9-._~:?#@!$&'()*+,;=/]{1,}))|(?!http)(?!https)([a-zA-Z0-9.]{1,})://)$",
+          onErrorMsg: "The value must be a valid URI with a trailing forward slash. HTTPS must be used instead of HTTP unless you are using localhost."
+        }
+      },
+      {
+        key: "oidcAppOIDCAttributesRequestMethod",
+        question: "Select your OIDC Attributes Request Method:",
+        type: "list",
+        map: "attributesRequestMethod",
+        required: true,
+        andConditions: [
+          {
+            key: "authProvidersUserPool",
+            value: "OIDC",
+            operator: "includes"
+          }
+        ]
+      },
+      {
+        key: "RemoveScopeConfirmation",
+        question: "Do you want to remove any auth scopes?",
+        type: "confirm",
+        andConditions: [
+          {
+            key: "authProvidersUserPool",
+            value: "OIDC",
+            operator: "includes"
+          },
+          {
+            key: "oidcAuthorizeScopes",
+            operator: "exists"
+          },
+          {
+                  key: 'updateFlow',
+                  value: 'providers',
+                  operator: '=',
+                }
+        ]
+      },
+      {
+        key: "RemoveScopes",
+        question: "Which scopes do you want to remove?",
+        type: "multiselect",
+        iterator: "oidcAuthorizeScopes",
+        andConditions: [
+          {
+            key: "oidcAuthorizeScopes",
+            operator: "exists"
+          },
+          {
+            onCreate: "never"
+          },
+          {
+            key: "RemoveScopeConfirmation",
+            value: true,
+            operator: "="
+          }
+        ]
+      },
+      {
+        key: "OIDCAdditionalScope",
+        question: "Do you want to add auth scopes?",
+        type: "confirm",
+        andConditions: [
+          {
+            key: "authProvidersUserPool",
+            value: "OIDC",
+            operator: "includes"
+          }
+        ]
+      },
+      {
+        key: "newOIDCAuthorizeScopes",
+        question: "Enter auth scope:",
+        required: true,
+        addAnotherLoop: "additional scope",
```

Comment:
"auth scope"
So the full prompt is "Do you want to add another auth scope" instead of "Do you want to add another additional scope".